### PR TITLE
Move example of slider to the proper place

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #704
[Problem] Bad place for slider element in examples app
[Solution] Change its place and edit proper files

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	</script>
+	<script src="../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="pageSlider">
+		<div class="ui-content ui-content-padding">
+			<input class="ui-slider" data-buttons="true" id="slider" max="10" name="circleSlider" type="range" />
+			<div class="ui-slider-icon" id="slider-icon" style="opacity: 0.5">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html]: Should the style be entered in html?
How about adding to a class conditionally in less?

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			<div class="ui-slider-title">
+				Slider name
+			</div>
+		</div>
+		<script src="extended_slider.js">
+		</script>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html]: check end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        </header>
+        <div class="ui-content">
+            <ul class="ui-list">
+                <li><a href="slider/index.html">Slider</a></li>
+            </ul>
+        </div>
+    </div>
+    
+</body>
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/index.html]: check end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider_subtitle.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+			<div class="ui-slider-subtitle">
+				Description
+			</div>
+		</div>
+		<script src="extended_slider.js">
+		</script>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider_subtitle.html]: check end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/slider.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+		</header>
+		<div class="ui-content ui-content-padding">
+			<input class="ui-slider" id="slider" max="10" name="circleSlider" type="range" />
+		</div>
+		<script src="slider.js">
+		</script>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/slider/slider.html]: check end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider_subtitle.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+	</script>
+	<script src="../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="pageSlider">
+		<div class="ui-content ui-content-padding">
+			<input class="ui-slider" data-buttons="true" id="slider" max="10" name="circleSlider" type="range" />
+			<div class="ui-slider-icon" id="slider-icon" style="opacity: 0.5">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider_subtitle.html]: ditto.

**Review by DohyungLim — COMMENTED** [review]
check to comment.

File: examples/wearable/UIComponents/contents/user-inputs-elements/index.html [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+<head>
+    <meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+    <title>
+        Wearable UI
+    </title>
+    <link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+    <link href="../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+    <link href="../../css/style.css" rel="stylesheet" />
+    <link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/user-inputs-elements/index.html]: media queries are no longer needed for OneUI 2.0 wearable

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html (hunk 1 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html]: @korneliakobiela @DohyungLim 
I think that other slider examples not refered in GUI Slider section could be moved to other section (see Style Guide section in GUI guideline) and we can take care of them later
we should have only just "Slider" example as it is in GUI

Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html]: @DohyungLim, The only things I've done is to move slider files from one place to another and add file 'user-inputs-elements/index.html'. These files have been reviewed some time ago. Is it really necessary to edit them?

**Review by lslachciak — APPROVED** [review]

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html (hunk 1 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html]: @lslachciak done

File: examples/wearable/UIComponents/contents/user-inputs-elements/slider/slider.html (hunk 5 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/slider/slider.html]: Done

File: examples/wearable/UIComponents/contents/user-inputs-elements/index.html (hunk 7 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/index.html]: Done

**Pull request merged**